on-off-board/github-org-management.md

```diff
@@ -0,0 +1,24 @@
+# GitHub Organization Management
+
+1. All new repositories and Teams should be created within the existing [NYPL](https://github.com/NYPL) Org, unless specifically called out in the Product Brief, BRD, or TAD. There should not be a need to create additional organizations outside of the NYPL Org or to add repositories within any of the other NYPL organizations, such as NYPL-Simplified. We are aiming to consolidate all extranneous orgs under the NYPL org.
+2. All active NYPL employees, this includes both FTEs and Contractors, who need Read, Write, or Admin access to NYPL repositories must be invited as a Member to the NYPL Org. They should _not_ be added to any repository as an Outside Collaborator.
+3. NYPL employees can use an existing GitHub account created with a non-NYPL email address or create a new GitHub account using their NYPL email address.
+4. It is recommended, but not required, that their [profile](https://github.com/settings/profile) include an easily recognizable name. This allows us to manage Members more easily.
```

Comment:
@nonword To accommodate folks who might not want to add their name to GitHub for reasons that they don't want to disclose to us, I made this optional. I think this is ok because the new teams structure allows us to see which portfolio a user is part of, and we'd be able to ask that portfolio tech lead or manager who the user is if we need to. Lmk what you think.

Reply:
We currently have two users who don't have their name listed and they're part of the `ereading-clients` team. I think me, Mike, and Russell know who they are so I'm not as concerned about needing to know who each person is by name when running through the full list of members. Because I can at least tell from a glance that they are part of the ereading team.